Documentation/CrossPlatform/UsingARFoundation.md

```diff
@@ -0,0 +1,92 @@
+# How to configure MRTK for iOS and Android [Experimental]
+
+## Install Required Packages
+
+1. Download and import the **Microsoft.MixedReality.Toolkit.Providers.UnityAR** package, from [GitHub](https://github.com/microsoft/MixedRealityToolkit-Unity/releases/tag/v2.2.0) or [NuGet](../MRTKNuGetPackage.md)
+
+> [!NOTE]
+> After you import the the UnityAR package, you will see the following error: 
+> *Assembly has reference to non-existent assembly 'Unity.XR.ARFoundation' (Assets/MixedRealityToolkit.Staging/UnityAR/Microsoft.MixedReality.Toolkit.Providers.UnityAR.asmdef)*
+". To resolve, install the correct version of ARFoundation listed below.
+
+
+2. In the Unity Package Manager (UPM), install the following packages:
+
+    ### Unity 2018.4.x 
+
+    | **Android** | **iOS** | Comments |
+    | --- | --- | --- |
+    | AR Foundation  <br/> Version: 1.5.0 - preview 6 | AR Foundation  <br/> Version: 1.5.0 - preview 6 | For Unity 2018.4, this package is included as a preview. To view package: Window > Package Manager > Advanced > Show Preview Packages|
+    | ARCore XR Plugin <br/> Version: 2.1.2 | | |
```

Comment:
This table formatted incorrectly for me in the preview

Reply:
The blank cells in the table were intentional to indicate that ARKit is not needed for Android and ARCore is not needed for iOS.  But if it looks like formatting issue then I can change the way the packages and versions are listed for each platform and unity version.

Reply:
This is what I see, 

![image](https://user-images.githubusercontent.com/168492/69734677-683fbf80-10fd-11ea-89c8-1bc70200bff2.png)

Ah, after staring at it for about 30 seconds more, I'm still not sure what the table is telling me :-) Would recommend maybe changing the visualization. Maybe just a bulletted list, one for android, one for iOS?